assets/templates/error/401.tmpl

```diff
@@ -0,0 +1,10 @@
+<div class="ons-page__container ons-container">
+    <div class="ons-grid ons-u-ml-no">
+        <div class="ons-grid__col ons-u-pl-no">
+            <h1 class="ons-u-fs-xxxl ons-u-mt-m ons-u-fw-b margin-top--8">{{ localise "401Title" .Language 1  }}</h1>
```

Comment:
It's part of the dp-design-system utils https://github.com/ONSdigital/dp-design-system/blob/main/src/scss/dp/overrides/helpers/_margins.scss

However there's margin styles in that file and margin styles in _utilities.scss file. Very messy 🙈 

Used it as it gave more margin/white space than the xxl margin from the design system. Although it's worth pointing out these error pages have been converted to design system styling without any design work prior. The plan is to ask design to look at them but that was outside the scope of this piece of work. Possibly worth leaving in for now and it can be addressed once the pages receive some design work from UX

Reply:
Fair enough, I'm happy to leave them in as the redesign to using the [design system error pages](https://service-manual.ons.gov.uk/design-system/patterns/error-status-pages) will ultimately remove it